content/post/performance-metrics-overview.md

```diff
@@ -2,7 +2,6 @@
 title = "Building low-overhead metrics collection for high-performance systems"
 date = 2018-02-20T07:20:00-05:00
 draft = false
-slug = "building-low-overhead-metrics-collection-for-high-performance-systems"
```

Comment:
as this wasn't be used before, i didn't want to leave it around in case it changed the URL. I'm pretty sure it wouldn't but, not reason to chance it.